.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+
+on:
+  pull_request:
+    branches:
+      - mc-develop
```

Comment:
How about we enable this for all PRs?
```suggestion
```

Reply:
We only use `mc-develop` for our purposes, and I didn't want to gate a sync with upstream on this CI passing.